suite-native/module-authorize-device/src/screens/passphrase/PassphraseFormScreen.tsx

```diff
@@ -69,7 +78,14 @@ export const PassphraseFormScreen = () => {
             }
         >
             <VStack spacing="medium">
-                <View style={applyStyle(animationWrapperStyle)}>
+                <View
+                    style={applyStyle(animationWrapperStyle)}
+                    onLayout={(event: LayoutChangeEvent) => {
```

Comment:
please remove this inline function and make it a named function above the UI code.

Reply:
7c5112727d